apps/server/src/toasts/toasts.service.ts

```diff
@@ -23,11 +26,21 @@ export class ToastsService {
     });
   }
 
+  getCurrentPeriod(currentDate: Date) {
+    const currentMonth = currentDate.getMonth();
+    const currentYear = currentDate.getFullYear();
+    const startDate =
+      currentMonth >= 0 && currentMonth < 5
```

Comment:
I believe here you can simply check if currentMonth is less than 5. Checking if it's greater than 0 isn't nesesary because getMonth can't return as far as I know negative numbers